librz/core/cmd/cmd_analysis.c

```diff
@@ -7403,6 +7240,56 @@ RZ_IPI RzCmdStatus rz_analysis_functions_merge_handler(RzCore *core, int argc, c
 	return RZ_CMD_STATUS_OK;
 }
 
+RZ_IPI RzCmdStatus rz_analysis_function_cc_set_get_handler(RzCore *core, int argc, const char **argv) {
+	RzAnalysisFunction *fcn = analysis_get_function_in(core->analysis, core->offset);
+	if (!fcn) {
+		return RZ_CMD_STATUS_ERROR;
```

Comment:
Probably worth to show error "no function at this offset" or something

Reply:
`analysis_get_function_in` does it